place_tracker/lib/place_util.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'place.dart';
+
+class PlaceUtil {
+  static BitmapDescriptor getPlaceMarkerIcon(PlaceCategory category) {
```

Comment:
Could be a private static on PlaceMap.

Maybe the switch should just determine the color...